macos/FluentUI/Button/Button.swift

```diff
@@ -128,6 +132,9 @@ open class Button: NSButton {
 			guard wantsLayer == true else {
 				preconditionFailure("wantsLayer must be set so that the image is rendered on the layer")
 			}
+			if image != nil {
+				addImage(forButtonState: ButtonState.normal, image: image!)
```

Comment:
don't use '!' through out this file

Reply:
The suggested swift pattern for this case is:

```
if let buttonImage = image {
    addImage(forButtonState: ButtonState.normal, image: buttonImage)
}
```